packages/map-template/src/components/Map/Map.jsx

```diff
@@ -203,6 +203,16 @@ function Map({ onLocationClick, onVenueChangedOnMap, useMapProviderModule, onMap
             mapView
         });
 
+        // Detect when the mouse hovers over a location and store the hovered location
+        // If the location is non-selectable, remove the hovering by calling the unhoverLocation() method.
```

Comment:
So the SDK applies hover effects to non-selectable Locations out of the box, and we chose to make the Map Template prevent that behaviour.

Seems like an odd way of slicing the cake: Shouldn't this be ported to the SDK? 

CC @markhougaard

Reply:
This was exactly the question that came to my mind as well, when I started implementing this. I had a talk with @matbmapspeople who confirmed the instructions were to implement this on the Map Template instead of the SDK. 

cc @markhougaard @ammapspeople